embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/spans/EmbraceSpan.kt

```diff
@@ -75,6 +76,16 @@ public interface EmbraceSpan {
         name: String
     ): Boolean = addEvent(name = name, timestampMs = null, attributes = null)
 
+    /**
+     * Add an [EmbraceSpanEvent] with the given [name] and [timestampMs]. Optionally, a set of attributes associated with the event can
+     * be passed in. Returns false if the Event was definitely not successfully added. Returns true if the validation at the Embrace
+     * level has passed and the call to add the Event at the OpenTelemetry level was successful.
+     */
+    public fun addEvent(
+        name: String,
+        timestampMs: Long?,
+    ): Boolean = addEvent(name = name, timestampMs = timestampMs, attributes = null)
```

Comment:
Does this make custom breadcrumbs somewhat redundant?

Reply:
Breadcrumb datasources should be able to leverage this directly if it's not already doing so, but I'm not adding new functionality that wasn't already there.